dist/toolkit.css

```diff
@@ -1,4 +1,1039 @@
-/*!
```

Comment:
if we are now including the css dist in the repo, could it contain a version number. i think it will be useful for debugging